fiftyone/utils/flash.py

```diff
@@ -11,15 +11,17 @@
 
 import numpy as np
 
-import fiftyone.core.metadata as fom
+import fiftyone.core.labels as fol
+import fiftyone.core.media as fom
 import fiftyone.core.utils as fou
 
-fou.ensure_import("flash>=0.4")
+fou.ensure_import("flash>=0.6dev")
```

Comment:
In the docs and PR description you said `0.7`, not `0.6dev`. Use `0.7` for consistency?

Reply:
I'm gonna go with 0.7dev here and in the docs so that it won't fail when using source builds of flash, sound good?

Reply:
👍